scripts/capture_resource_metrics.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# PREREQUISITES
+# - jq
+# - oc (logged in at the cmd line in order to get the bearer token)
+# - a "perf-test-start-time.txt" file with a valid rfc3339 timestamp from a moment before performance tests started 
+#   (it can be produced with this multiplatform command: `date -u +%Y-%m-%dT%TZ > perf-test-start-time.txt`)
+# - a "perf-test-end-time.txt" file with a valid rfc3339 timestamp from a moment after performance tests finished
```

Comment:
Expected timestamps look like this: `2020-10-29T09:22:00Z`
We could switch just to timestamp in seconds if needed.
I am open to suggestions on how to best create this timestamp files, e.g.: it could be part of the test suite.